sdk/cosmos/azure-cosmos/azure/cosmos/container.py

```diff
@@ -42,20 +42,18 @@
 
 
 class ContainerProxy(object):
-    """
-    An interface to interact with a specific DB Container.
-    This class should not be instantiated directly, use :func:`DatabaseProxy.get_container_client` method.
+    """An interface to interact with a specific DB Container.
+
+    This class should not be instantiated directly. Instead, use the
+    :func:`DatabaseProxy.get_container_client` method to get an existing
+    container, or the :func:`Database.create_container` method to create a
+    new container.
 
-    A container in an Azure Cosmos DB SQL API database is a collection of documents,
-    each of which represented as an Item.
+    A container in an Azure Cosmos DB SQL API database is a collection of
+    documents, each of which represented as an Item.
```

Comment:
each of which _is_ represented as an Item.

Reply:
fixed in https://github.com/Azure/azure-sdk-for-python/pull/8607/commits/c14bb0d8adcb5e63ac1d6a2fab3819ccca50707e